chia/rpc/wallet_rpc_api.py

```diff
@@ -1186,13 +1187,13 @@ async def did_get_recovery_list(self, request):
         wallet_id = int(request["wallet_id"])
         wallet: DIDWallet = self.service.wallet_state_manager.wallets[wallet_id]
         recovery_list = wallet.did_info.backup_ids
-        recover_hex_list = []
+        recover_dids_list = []
         for _ in recovery_list:
```

Comment:
nit: Can we use a variable name here? I expect that the original code was ignoring the value, but since we're actually passing it into a function, probably best to give it a name.